.github/workflows/ipa-clone-test.yml

```diff
@@ -503,6 +504,7 @@ jobs:
           #   TODO: remove this change once the following PR is merged:
           #   https://github.com/freeipa/freeipa/pull/6971
           sed -e 's/^\(ca.crl.MasterCRL.enable\)=.*$/\1=false/' \
+              -e '/ca.certStatusUpdateInterval=0/d' \
               CS.cfg.primary.after-crl-update \
               | sort > actual
```

Comment:
Instead of removing this param from the actual result (i.e. not validating the param) could we add this into the expected result so we can confirm that the param was indeed modified as expected?

```
-e '$ a ca.certStatusUpdateInterval=0' \
```